bash_scripts/bash_scripts.md

```diff
@@ -70,7 +70,22 @@ If you are using a computer running macOS (i.e. an Apple computer) you can use t
 
 <div class = "important">
 <b style="color: rgb(var(--color-highlight));">Important note</b><br>
+
 We want to be able to search, move, and rename files during this module, but don't want to do that with your important files. Therefore we will set up a little directory with a few files to experiment with. You can safely delete the whole thing afterwards if you want.
+
+
+The `learning_bash` folder contains a few different types of files and folders for us to experiment with. The ones we will be using in this module are:
```

Comment:
same comments as before -- move this to where files are actually starting to be worked with